test_refund_tickets_details.py

```diff
@@ -0,0 +1,71 @@
+import refund_tickets_details
+import movie_theatre
+
+def customer_db():
+    """Define Customer database for testing"""
+    return {
+        "Bob" : {"movies":[],
+                 "cash" : 100.0},
+        "Jim" : {"movies": ["Xmen 8: The Xmennening"],
+                 "cash" : 10.0},
+        "Cary" : {"movies": ["Gigli: The Play: The Book: The movie", "The Bromance"],
+                  "cash" : 120.0},
+        "Ricci": {"movies": [],
+                  "cash" : 4.0}
+    }
+
+def movie_db():
```

Comment:
refactor out this function and the customer db function into a test_helpers.py module and import that into each of these test files